packages/rest-typings/src/v1/users/UsersSetPreferenceParamsPOST.ts

```diff
@@ -42,6 +45,7 @@ export type UsersSetPreferencesParamsPOST = {
 		dontAskAgainList?: Array<{ action: string; label: string }>;
 		featuresPreview?: { name: string; value: boolean }[];
 		themeAppearence?: ThemePreference;
+		fontSize?: FontSize;
```

Comment:
Can we add this `fontSize` property here: https://github.com/RocketChat/Rocket.Chat/blob/develop/apps/meteor/server/methods/saveUserPreferences.ts#L7 and here: https://github.com/RocketChat/Rocket.Chat/blob/develop/apps/meteor/server/methods/saveUserPreferences.ts#L51 as well?